app/models/custom_field.rb

```diff
@@ -73,7 +73,8 @@ def uniqueness_of_name_with_scope
 
   validate :validate_default_value_in_translations
 
-  validate :validate_name
+  # FIXME: Rails 4/Globalize 4: clean-up CustomField validations
+  # validate :validate_name
```

Comment:
Commenting this out (in 7ec22ec) was probably the wrong move – it took us from 2 failing specs to 4 :( 

* `./spec/models/work_package_spec.rb`: https://travis-ci.org/opf/openproject/jobs/60909624
* `./spec/controllers/custom_fields_controller_spec.rb`: https://travis-ci.org/opf/openproject/jobs/60935923

@slawa85 I believe you introduced this method (a2cf6871) – any ideas here?

@NobodysNightmare @Azure7111 – since you also know your way around the `CustomField` code, perhaps you'd also be interested in taking a look?